doc/5.reference/expr-help.pd

```diff
@@ -1,4 +1,4 @@
-#N canvas 275 90 942 546 12;
```

Comment:
the changes to expr-help are much more than just adding the forgotten "%".
since the help-patch is rather deeply nested, it's hard to spot all the changes. what else did you change?

Reply:
that's the only information I changed, the other changes are just reorganizing the patch, shifting a bit to the right and left, window size, things like that. Note I authored all the big recent changes in this help file patch, I wouldn't really wanna change anything.